jupyter/datascience/anaconda-python-3.8/environment.yml

```diff
@@ -0,0 +1,356 @@
+name: /opt/app-root
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - _openmp_mutex=5.1=1_gnu
+  - _tflow_select=2.3.0=mkl
+  - abseil-cpp=20211102.0=h27087fc_1
+  - absl-py=1.4.0=pyhd8ed1ab_0
+  - adal=1.2.7=pyhd8ed1ab_0
+  - aiohttp=3.8.1=py38h0a891b7_1
+  - aiosignal=1.3.1=pyhd8ed1ab_0
+  - ansiwrap=0.8.4=py_0
+  - anyio=3.6.2=pyhd8ed1ab_0
+  - argon2-cffi=21.3.0=pyhd8ed1ab_0
+  - argon2-cffi-bindings=21.2.0=py38h0a891b7_2
+  - astroid=2.15.5=py38h578d9bd_0
+  - asttokens=2.2.1=pyhd8ed1ab_0
+  - astunparse=1.6.3=pyhd8ed1ab_0
+  - async-timeout=4.0.2=pyhd8ed1ab_0
+  - attrs=23.1.0=pyh71513ae_1
+  - autopep8=2.0.2=pyhd8ed1ab_0
+  - babel=2.12.1=pyhd8ed1ab_1
+  - backcall=0.2.0=pyh9f0ad1d_0
+  - backports=1.0=pyhd8ed1ab_3
+  - backports.functools_lru_cache=1.6.4=pyhd8ed1ab_0
+  - beautifulsoup4=4.12.2=pyha770c72_0
+  - black=23.3.0=py38h578d9bd_1
+  - blas=1.0=mkl
+  - bleach=6.0.0=pyhd8ed1ab_0
+  - blinker=1.6.2=pyhd8ed1ab_0
+  - bokeh=3.1.1=pyhd8ed1ab_0
+  - bottleneck=1.3.5=py38h7deecbd_0
+  - brotli=1.0.9=h166bdaf_7
+  - brotli-bin=1.0.9=h166bdaf_7
+  - c-ares=1.19.0=h5eee18b_0
+  - ca-certificates=2023.5.7=hbcca054_0
+  - cached-property=1.5.2=hd8ed1ab_1
+  - cached_property=1.5.2=pyha770c72_1
+  - cachetools=5.3.0=pyhd8ed1ab_0
+  - certifi=2023.5.7=pyhd8ed1ab_0
+  - cffi=1.15.0=py38h3931269_0
+  - charset-normalizer=2.0.4=pyhd3eb1b0_0
+  - click=8.1.3=unix_pyhd8ed1ab_2
+  - cloudpickle=2.2.1=pyhd8ed1ab_0
+  - colorama=0.4.6=pyhd8ed1ab_0
+  - configparser=5.3.0=pyhd8ed1ab_0
+  - contourpy=1.0.5=py38hdb19cb5_0
+  - cryptography=37.0.2=py38h2b5fc30_0
+  - cycler=0.11.0=pyhd8ed1ab_0
+  - dbus=1.13.18=hb2f20db_0
+  - decorator=5.1.1=pyhd8ed1ab_0
+  - defusedxml=0.7.1=pyhd8ed1ab_0
+  - deprecated=1.2.13=pyh6c4a22f_0
+  - deprecation=2.1.0=pyh9f0ad1d_0
+  - dill=0.3.6=pyhd8ed1ab_1
+  - docstring-to-markdown=0.12=pyhd8ed1ab_0
+  - docstring_parser=0.15=pyhd8ed1ab_0
+  - elyra=3.15.0=pyhd8ed1ab_0
```

Comment:
If this is just for the ODH community, then we are good with this
if this would be included in the RHODS,
please install each plugin separately so we have control.
https://github.com/opendatahub-io/notebooks/blob/main/jupyter/datascience/ubi8-python-3.8/Pipfile#L10-L13

Reply:
So, funny story, there isn't any more granular package, that's it. I can always just remove elyra stuff if we want

Reply:
Nvm, there is a more granular one, ok, I'll swap that